### PR TITLE
Multiple Edits

### PR DESCRIPTION
**Pull request by LuncBurner**
Remove migration guide from home page cards.  Also, this should be removed from the main page navigation menu.

### CODE REVIEW

File: docs/learn/protocol.mdx [hunk 1]
```diff
@@ -116,55 +70,54 @@ Staked Luna never leaves the possession of the delegator. Even though it can’t
...
 - **Unbonding**: Luna that is in the process of becoming unbonded from a validator and does not accrue rewards. This process takes 21 days to complete.
 
 ### Bonding, staking, and delegating
 
-Generally, the terms bonding, staking, and delegating can be used interchangeably, as they happen in the same step. A delegator delegates Luna to a validator, the Luna gets bonded to the validator, and the bonded Luna gets added to the validator's stake.
+Generally, the terms bonding, staking, and delegating can be used interchangeably, as they happen in the same step. A delegator delegates Lunc to a validator, the Lunc gets bonded to the validator, and the bonded Lunc gets added to the validator’s stake.
 
-Delegators can bond Luna to any validator in the [active set](./glossary.mdx#active-set) using the delegate function in Station. Delegators start earning staking rewards the moment they bond or stake to a validator.
+Delegators can bond Luna to any validator in the [active set](https://classic-docs.terra.money/docs/learn/glossary.html#active-set) using the delegate function in Terra Station. Delegators start earning staking rewards the moment they bond or stake to a validator.
```
Comment (alchemist-ti) [inline docs/learn/protocol.mdx]:
```suggestion
Delegators can bond Luna to any validator in the [active set](./learn/glossary.html#active-set) using the delegate function in Terra Station. Delegators start earning staking rewards the moment they bond or stake to a validator.
```

File: docs/learn/protocol.mdx [hunk 2]
```diff
@@ -2,92 +2,46 @@ import Admonition from '@theme/Admonition';
...
 
-Validators are the miners of the Terra [blockchain](./glossary.mdx#blockchain). They are responsible for securing the Terra blockchain and ensuring its accuracy. Validators run programs called full nodes which allow them to verify each transaction made on the Terra network. Validators propose blocks, vote on their validity, and add each new block to the chain in exchange for staking rewards from transaction fees. Users can stake their Luna to validators in exchange for staking rewards. Validators also play an important role in the governance of the Terra protocol.
+Validators are the miners of the Terra [blockchain](https://classic-docs.terra.money/docs/learn/glossary.html#blockchain). They are responsible for securing the Terra blockchain and ensuring its accuracy. Validators run programs called full nodes which allow them to verify each transaction made on the Terra network. Validators propose blocks, vote on their validity, and add each new block to the chain in exchange for staking rewards from transaction fees. Users can stake their Luna to validators in exchange for staking rewards. Validators also play an important role in the governance of the Terra protocol.
```
Comment (alchemist-ti) [inline docs/learn/protocol.mdx:37]: `Terra [blockchain](./learn/glossary.html#blockchain)`

**Review by alchemist-ti — COMMENTED** [review]
LGTM, it is better to use relative paths for some links, so that it can be more friendly on github.

**Comment by LuncBurner** [comment]
@blendsdk Made a bunch more edits, can you review and merge accordingly?

**Pull request merged**